tests/pairlist/test_pairlist.py

```diff
@@ -221,7 +225,8 @@ def test_VolumePairList_whitelist_gen(mocker, whitelist_conf, shitcoinmarkets, t
             assert log_has_re(r'^Removed .* from whitelist, because stop price .* '
                               r'would be <= stop limit.*', caplog)
         if pairlist['method'] == 'PriceFilter':
-            assert log_has_re(r'^Removed .* from whitelist, because 1 unit is .*%$', caplog)
+            assert (log_has_re(r'^Removed .* from whitelist, because 1 unit is .*%$', caplog) or
+                    log_has_re(r"^Removed .* from whitelist, because 'last' is empty.*", caplog))
```

Comment:
"...because ticker,last is empty" ?
"...because ticker['last'] is empty" ?
"...because 'last' in ticker is empty" ?